### DISCUSSION TITLE
Hide icon-search on individual column

### DISCUSSION THREAD

**Discussion by wit3** [filamentphp/filament#3988] [Ideas]
Hi,
It's possible to hide icon in search individual talbe column?
'cause i have a column with label that has a  short lenght, ex: "Id" the textinput is hidden by icon.

PS: it' also possible to change the time of deferring input in this individual serach ?

**Comment by Sicklou** [comment]
I've got the same issue.
For thin columns, the input is not visible, we could gain some places by getting rid of the icon.

Before
![image](https://user-images.githubusercontent.com/8366863/214572022-6a459952-8dee-4ea9-a4cd-30cecb7ed11d.png)
After
![image](https://user-images.githubusercontent.com/8366863/214572244-6eab90b0-4e95-483d-b9cb-2c9ac0691dea.png)

**Reply by fikurimax** [reply]
any updates? or how do you get rid of it?

**Reply by Sicklou** [reply]
I didn't :)
